python/tvm/script/tir/__init__.pyi

```diff
@@ -299,7 +284,7 @@ special_stmt - Annotations
 """
 
 def buffer_var(dtype: str, storage_scope: str) -> Var: ...
-def func_attr(attrs: Mapping[str, Object]) -> None: ...
+def func_attr(attrs: Mapping[str, Object | str | bool | int | float]) -> None: ...
```

Comment:
`|` syntax is only available in python 3.10, we may prefer using `Union` for now

Reply:
Huh, it worked with my python 3.8. But I've switched this to a `Union`